core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -576,7 +576,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
         if (requestHandlerPool != null)
           CoreUtils.swallow(requestHandlerPool.shutdown(), this)
 
-        CoreUtils.swallow(kafkaScheduler.shutdown(), this)
+        if (kafkaScheduler != null) CoreUtils.swallow(kafkaScheduler.shutdown(), this)
```

Comment:
Nit: please move the `swallow` to the next line to be consistent with the rest of the code.

Reply:
> Nit: please move the swallow to the next line to be consistent with the rest of the code.

copy that. Thanks for the reviews.